config/manifests/bases/awx-operator.clusterserviceversion.yaml

```diff
@@ -4,6 +4,12 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
+    categories: Integration & Delivery
+    containerImage: quay.io/ansible/awx-operator:devel
+    createdAt: "2019-10-30"
```

Comment:
What's this date about?

Reply:
@shanemcd "2019-10-30" is when this `awx-operator` repo was created and the first commit was made!
This is a required field on OperatorHub.io.